.github/workflows/test-native.yml

```diff
@@ -81,27 +75,23 @@ jobs:
         run: |
           cd ${GITHUB_WORKSPACE}/presto-native-execution/_build/debug
           # Delete static libraries
-          rm `find ./ -name lib\*.a` 
-
-      - name: Cache local Maven repository
-        id: cache-maven
-        uses: actions/cache@v3
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-2-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-2-
-
-      - name: Populate maven cache
-        if: steps.cache-maven.outputs.cache-hit != 'true'
-        run: ./mvnw de.qaware.maven:go-offline-maven-plugin:resolve-dependencies
+          rm `find ./ -name lib\*.a`
 
       - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./mvnw install ${MAVEN_FAST_INSTALL} -pl '!presto-docs,!presto-server,!presto-server-rpm,!presto-test-coverage'
+          mvn clean install ${MAVEN_FAST_INSTALL} -pl 'presto-native-execution' -am
 
       - name: Run e2e tests
         run: |
           rm -rf /tmp/hive_data/tpch/
```

Comment:
Can you add a comment here to explain why we are doing this?

Also, please, update commit message to describe the problem and the solution as well as limitations.

@majetideepak Deepak, would you help review this change?

Reply:
> Can you add a comment here to explain why we are doing this?
> 
> Also, please, update commit message to describe the problem and the solution as well as limitations.
> 
> @majetideepak Deepak, would you help review this change?

I use "-am" here to only build only dependent component which would save us some time. I will update the pr